libcxx/test/std/input.output/iostream.format/output.streams/ostream.formatted/ostream.formatted.print/locale-specific_form.pass.cpp

```diff
@@ -0,0 +1,2200 @@
+//===----------------------------------------------------------------------===//
```

Comment:
It's taken from `test/std/utilities/format/format.functions/locale-specific_form.pass.cpp`. Since they reside in completely different directories I'm not sure whether there is a good way to share these. That's why I copied the file.